.claude/agents/reset-sync-upstream.md

```
@@ -0,0 +1,63 @@
+---
+name: reset-sync-upstream
+description: Fetch upstream changes without merging. Update remote tracking refs.
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)